### DISCUSSION TITLE
pemfile.NewProvider({   5h0m0s}) grpc.s3 failed: pemfile: at least one credential file needs to be specified

### DISCUSSION THREAD

**Discussion by mlacko64** [seaweedfs/seaweedfs#4215] [General]
Hello,
there is following message printed when weed starts:

W0216 08:46:51.884350 tls.go:39 pemfile.NewProvider({   5h0m0s}) grpc.s3 failed: pemfile: at least one credential file needs to be specified

I noticed it also in other discussions outputs. I am using latest Seaweedfs version.

Is it something which should be configured or fixed on my side?

**Comment by chrislusf** [comment]
No worry. it is just a warning in case the TLS is configured.

**Reply by mlacko64** [reply]
Thank you very much for your reply
